packages/components/collapsible-panel/src/collapsible-panel.js

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import isNil from 'lodash/isNil';
+import uniqueId from 'lodash/uniqueId';
```

Comment:
Is this a new dep? Maybe using uuid can help. We use it everywhere else. Not sure, just a thought.

Reply:
`uuid` isn't a dependency in UI-kit either.

However, since it's used in the MC it might be preferable to use the same tool 👍

Reply:
On the other hand UIKit uses lodash and the MC is not the only user. Sorry, the discussion I started is a bit side-tracking. Lodash uniqueid is fine. Proceed 🤓.

Reply:
Yeah, also, I was thinking again and I prefer the simple sequential `id` from `lodash/uniqueId` rather than the much longer `uuid`, for HTML elements.